src/Symfony/Component/Pgp/Tests/Crypto/PgpEncryptorTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Pgp\Tests\Crypto;
```

Comment:
the namespace should match the namespace of the test code, which is not in a `Crypto` subnamespace.